src/test/java/ch/liquidmind/inflection/test/blackbox/BlackboxTest.java

```diff
@@ -0,0 +1,101 @@
+package ch.liquidmind.inflection.test.blackbox;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import ch.liquidmind.inflection.compiler.util.InflectionCompilerTestUtility;
+import ch.liquidmind.inflection.model.external.Taxonomy;
+import ch.liquidmind.inflection.model.external.View;
+import ch.liquidmind.inflection.model.external.util.TaxonomyTestUtility;
+import ch.liquidmind.inflection.proxy.Proxy;
+import ch.liquidmind.inflection.proxy.ProxyRegistry;
+import ch.liquidmind.inflection.proxy.util.ProxyGeneratorTestUtility;
+import ch.liquidmind.inflection.test.TestUtility;
+import ch.liquidmind.inflection.test.model.B1;
+import ch.liquidmind.inflection.test.model.TestEnum;
+
+public class BlackboxTest
```

Comment:
The main issue here is that we should be testing the grammar by looking at the compiled/linked taxonomy directly, not the proxies that are generated from them. The proxies are a separate issue, but I want to first make sure that the "pure" taxonomies are actually working properly first. This will have an effect on your utility classes, since you are currently combining taxonomy loading/generation issues in one place (ProxyGeneratorTestUtility); this will need to be split into two classes.